lib/MetaCPAN/Model/Search.pm

```diff
@@ -0,0 +1,484 @@
+package MetaCPAN::Model::Search;
+
+use Moose;
```

Comment:
The only reason I can think of is that I didn't know about it (also, I copied from MetaCPAN::Model::Release which does't use it).